src/forms/ThresholdsSelection.tsx

```diff
@@ -92,120 +91,134 @@ class ThresholdsSelectionInput extends Component<MyProps & InjectedIntlProps, My
         const { comparison, thresholds } = value;
 
         return (
-            <div>
-                <div className={styles.AddThreshold}>
-                    <label htmlFor="comparison" className={styles.Comparision}>
-                        <FormattedMessage
-                            id="notifications_app.comparison"
-                            defaultMessage="Comparison"
-                        />
-                    </label>
-                    <div className={styles.ThresholdInput}>
-                        <div className={styles.ThresholdValueInput}>
+            <table>
```

Comment:
I notice you use a table here.
I am myself not a fan of the table, but this is a good use-case for it.
I am interested why did you choose table over div with flexbox?

Reply:
hmm I am not the one who comes up with the idea of using table here but Remco :) and I also want to try it with table and it works well I think. I think it's actually better than using div with flexbox here.